app/src/main/java/org/jellyfin/androidtv/ui/playback/overlay/action/FastForwardAction.kt

```diff
@@ -0,0 +1,12 @@
+package org.jellyfin.androidtv.ui.playback.overlay.action
+
+import android.content.Context
+import androidx.leanback.widget.PlaybackControlsRow
+import org.jellyfin.androidtv.ui.playback.overlay.VideoPlayerAdapter
+
+class FastForwardAction(context: Context) : PlaybackControlsRow.FastForwardAction(context),
+	AndroidAction {
```

Comment:
I'd format it like this:

```suggestion
class FastForwardAction(
	context: Context
) : PlaybackControlsRow.FastForwardAction(context), AndroidAction {
```

or just a single line

```suggestion
class FastForwardAction(context: Context) : PlaybackControlsRow.FastForwardAction(context), AndroidAction {
```

This also applies to the other actions.